source/wp-content/themes/wporg-gutenberg/functions.php

```diff
@@ -37,253 +37,258 @@ function () {
 	 */
 	function gutenberg_editor_scripts_and_styles( $hook ) {
 
-		// Enqueue heartbeat separately as an "optional" dependency of the editor.
-		// Heartbeat is used for automatic nonce refreshing, but some hosts choose
-		// to disable it outright.
-		wp_enqueue_script( 'heartbeat' );
-
-		global $post;
-
-		// Set initial title to empty string for auto draft for duration of edit.
-		// Otherwise, title defaults to and displays as "Auto Draft".
-		$is_new_post = 'auto-draft' === $post->post_status;
-
-		// Set the post type name.
-		$post_type        = get_post_type( $post );
-		$post_type_object = get_post_type_object( $post_type );
-		$rest_base        = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
-
-		$preload_paths = array(
-			'/',
-			'/wp/v2/types?context=edit',
-			'/wp/v2/taxonomies?per_page=-1&context=edit',
-			'/wp/v2/themes?status=active',
-			sprintf( '/wp/v2/%s/%s?context=edit', $rest_base, $post->ID ),
-			sprintf( '/wp/v2/types/%s?context=edit', $post_type ),
-			sprintf( '/wp/v2/users/me?post_type=%s&context=edit', $post_type ),
-			array( '/wp/v2/media', 'OPTIONS' ),
-			array( '/wp/v2/blocks', 'OPTIONS' ),
-		);
-
-		/**
-		 * Preload common data by specifying an array of REST API paths that will be preloaded.
-		 *
-		 * Filters the array of paths that will be preloaded.
-		 *
-		 * @param array $preload_paths Array of paths to preload
-		 * @param object $post         The post resource data.
-		 */
-		$preload_paths = apply_filters( 'block_editor_preload_paths', $preload_paths, $post );
-
-		// Ensure the global $post remains the same after
-		// API data is preloaded. Because API preloading
-		// can call the_content and other filters, callbacks
-		// can unexpectedly modify $post resulting in issues
-		// like https://github.com/WordPress/gutenberg/issues/7468.
-		$backup_global_post = $post;
-
-		$preload_data = array_reduce(
-			$preload_paths,
-			'rest_preload_api_request',
-			array()
-		);
+		if ( wp_is_mobile() ) {
```

Comment:
Make this condition inside `gutenberg_editor_scripts_and_styles` since
1. We're still rendering Gutenberg editor but in mobile version.
2. Styles can be shared without duplicating them.